tdx-attestation-app/src/HttpClient.cpp

```diff
@@ -51,6 +51,11 @@ HttpClientResult HttpClient::InvokeHttpRequest(std::string& http_response,
     // Set SSL versions to be used
     curl_easy_setopt(curl_handle, CURLOPT_SSLVERSION, CURL_SSLVERSION_TLSv1_2);
 
+    // SUSE Linux uses different SSL cert path than Ubuntu
```

Comment:
Do we need a check here to make sure this is not run for ubuntu?

Reply:
Not really Ubuntu has different cert name and path. The Library is only compiled in Debian which has the same path as Ubuntu, but SUSE is different. This actually works in both. Once there is a way to compile the library on different Linux flavors, this can be removed and the we just recompile in a SUSE environment